src/elements/content-preview/stories/tests/ContentPreview-visual.stories.js

```diff
@@ -143,7 +143,7 @@ export const citationDisabled = {
         const answer = await modal.findByText('Public APIs are important because of key and important reasons.');
         expect(answer).toBeInTheDocument();
 
-        expect(modal.getByText('Based on:')).toBeInTheDocument();
+        expect(modal.queryByText('Based on:')).not.toBeInTheDocument();
```

Comment:
where is the text `'Based on:'` actually rendered from? does it come from the api response or is it hardcoded in the repo?

Reply:
part of the citation feature that appears in the answer if the citations are present via the shared-features